wiki/BanchoBot/de.md

```diff
@@ -1,25 +1,158 @@
-| :-- | :-- | :-- | :-- | :-- |
-| `HELP` | Keine | Zeigt alle verfügbaren Befehle. | `!help` | Diese Liste |
-| `ROLL` | `[nummer]` | Gibt eine zufällige Zahl bis maximal `nummer` zurück (standardmäßig 100). | `!roll 1000` | "Nathanael rolls 789 point(s)" |
-| `STATS` | `<nutzername>` | Zeigt Statistiken über den angegebenen Spieler an. Abhängig vom aktuellen Spielmodus. | `!stats peppy` | "Stats for [peppy](https://osu.ppy.sh/users/2): Score: 412,018,739 (#94718), Plays: 7073 (lv65), Accuracy: 87.31%" |
-| `WHERE` | `<nutzername>` | Zeigt den aktuellen Ort des Spielers. | `!where Kyubey` | "Kyubey is in Russia" |
-| `FAQ` | `[sprache:]<eintrag>/list` | Zeigt den angegebenen Eintrag. Verwende `list` als Eintrag, um alle verfügbaren Einträge anzuzeigen. | `!faq ping` | "Pong!" |
-| `REPORT` | `<nutzername> <grund>` | [Meldet](/wiki/Reporting_bad_behaviour) einen Spieler bei den Chatmoderatoren. | `!report S_o_h spamming in #japanese` | "Chat moderators have been alerted. Thanks for your help." |
-| `REQUEST` | Keine | Erhalte eine zufällige Beatmap, bei der vom Ersteller Hilfe durch [Modding](/wiki/Modding) angefordert wurde. | `!request` | "[HoneyWorks - Tokyo Summer Session feat. CHiCO](https://osu.ppy.sh/beatmapsets/426252) by MrSergio" |
+*Für eine Liste der Client-Befehle im Spiel, siehe: [Chat-Konsole](/wiki/Client/Interface/Chat_console#commands-list)*
+
+BanchoBot kann durch spezifische getippte Nachrichten im Chat auf Spielerbefehle reagieren. Alle BanchoBot-Befehle beginnen mit einem Ausrufezeichen (`!`), gefolgt von dem Namen des Befehls (ohne Leerzeichen dazwischen). Diese Befehle können in Chat-Kanälen und in privaten Nachrichten mit BanchoBot verwendet werden.
+
+Wenn ein normaler Benutzer einen Befehl in einem öffentlichen Chat-Kanal sendet, sehen andere Benutzer ihn nicht, und die Antwort wird in einer privaten Nachricht mit BanchoBot angezeigt. Benutzer können auch den Befehl `/bb` im Spiel-Client verwenden, um automatisch einen Tab mit BanchoBot zu öffnen und den Befehl sofort zu senden.
+
+Eine Liste aller BanchoBot-Befehle befindet sich unten:
+
+- [Help](#help)
+- [Roll](#roll)
+- [Stats](#stats)
+- [Where](#where)
+- [FAQ](#faq)
+- [Report](#report)
+
+### Help
+
+```
+!help
+```
+
+`!help` zeigt eine Liste aller verfügbaren BanchoBot-Befehle an. Ein Beispiel für das Senden dieses Befehls wird unten gezeigt:
+
+```
+13:00 pippi: !help
+13:00 BanchoBot: Standard Commands (!COMMAND or /msg BanchoBot COMMAND):
+13:00 BanchoBot: WHERE <user>
+13:00 BanchoBot: STATS <user>
+13:00 BanchoBot: FAQ <item>|list
+13:00 BanchoBot: REPORT <reason> - call for an admin
+13:00 BanchoBot: REQUEST [list] - shows a random recent mod request
+13:00 BanchoBot: ROLL <number> - roll a dice and get random result from 1 to number(default 100)
+```
+
+<!--note for editors: the code block above reflects the exact response from banchobot -->
+
+*Tipp: Der Befehl `!request` wird von BanchoBot nicht mehr unterstützt.
```

Comment:
```suggestion
*Hinweis: Der Befehl `!request` wird von BanchoBot nicht mehr unterstützt.
```
ASC uses `Hinweis` for `Notice`. `Tipp` is `Note`.

Reply:
my bad, i actually checked ASC for that but i guess i confused Note and Notice 😄